ppl/zqd/storage/filestore/filestore.go

```diff
@@ -230,39 +220,11 @@ func (s *Storage) syncInfoFile() error {
 	info := info{
 		MinTime:       s.span.Ts,
 		MaxTime:       s.span.End(),
-		AlphaMigrated: s.alphaMigrated,
+		AlphaMigrated: true,
```

Comment:
You can remove the `AlphaMigrated` field from the `info` type definition, as there won't be anything paying attention to it.

Reply:
I wasn't sure if the app would do something...